examples/nextjs-dashboard/src/liveblocks.config.ts

```diff
@@ -0,0 +1,26 @@
+import { createClient } from "@liveblocks/client";
+import { createRoomContext } from "@liveblocks/react";
+
+const client = createClient({
+  publicApiKey: process.env.NEXT_PUBLIC_LIVEBLOCKS_PUBLIC_KEY!,
+});
+
+export type CursorPosition = {
```

Comment:
```suggestion
type CursorPosition = {
```
After changes we don't need to export this anymore